menu.js

```diff
@@ -20,3 +21,195 @@ navMenu.addEventListener('click', () => {
+  btn.textContent = 'See Projects';
+  btn.addEventListener('click', () => {
+    if (window.screen.width > 992) {
+      languagesUlLi4.style.display = 'inline';
+      languagesUlLi5.style.display = 'inline';
+      languagesUlLi6.style.display = 'inline';
+    }
+    const popupWindowMainCont = document.createElement('section');
+    workSectionContainer.appendChild(popupWindowMainCont);
+    popupWindowMainCont.className = 'popupWindowCont';
+    const popupwindow = document.createElement('section');
+    popupWindowMainCont.appendChild(popupwindow);
+    popupwindow.className = 'popupWindow';
+    const closeIcon = document.createElement('img');
+    closeIcon.src = 'images/xIcon.svg';
+    closeIcon.style.float = 'right';
+    popupwindow.appendChild(closeIcon);
+    closeIcon.addEventListener('click', () => {
+      languagesUlLi4.style.display = 'none';
+      languagesUlLi5.style.display = 'none';
+      languagesUlLi6.style.display = 'none';
+      popupWindowMainCont.style.display = 'none';
+      bgdiv.appendChild(h2);
+      bgdiv.appendChild(bgdivUl);
+      cardSection.appendChild(featuredImage);
+      bgdiv.appendChild(bgdivP);
+      bgdivP.textContent = popupObject[i].description;
+      bgdiv.appendChild(languagesUl);
+      bgdiv.appendChild(btn);
+      btn.style.width = '130px';
+      languagesUl.style.marginBottom = '0';
+    });
+    popupwindow.appendChild(h2);
+    popupwindow.appendChild(bgdivUl);
+    popupwindow.appendChild(featuredImage);
+    const pDivider = document.createElement('div');
+    pDivider.className = 'pDivider';
+    popupwindow.appendChild(pDivider);
+    pDivider.appendChild(bgdivP);
+    bgdivP.textContent = popupObject[i].popupDescription;
+    const techDiv = document.createElement('div');
+    techDiv.className = 'techDiv';
+    pDivider.appendChild(techDiv);
+    techDiv.appendChild(languagesUl);
+    const popupbtn1 = document.createElement('a');
+    const popupbtn2 = document.createElement('a');
+    const popupbtnDiv = document.createElement('div');
+    techDiv.appendChild(popupbtnDiv);
+    popupbtn1.className = 'popupClickme1';
+    popupbtn2.className = 'popupClickme2';
+    popupbtn1.textContent = 'See live';
+    popupbtn2.textContent = 'See source';
+    const icon1 = document.createElement('img');
+    const icon2 = document.createElement('img');
+    icon2.src = 'images/githubinbutton.svg';
+    popupbtn2.appendChild(icon2);
+    icon1.src = 'images/roundShape.png';
+    icon1.style.marginBottom = '-3px';
+    icon2.style.margin = '0 5px -3px 6px';
+    icon1.style.margin = '0 5px -3px 6px';
+    popupbtn1.appendChild(icon1);
+    popupbtn2.href = popupObject[i].secodbutton;
+    popupbtnDiv.append(popupbtn1, popupbtn2);
+    const hr = document.createElement('hr');
+    hr.className = 'hr';
+    techDiv.insertBefore(hr, popupbtnDiv);
+    hr.style.color = '#ebecf0';
+    popupbtnDiv.className = 'popupBtnDiv';
+    languagesUl.style.marginBottom = '20px';
```

Comment:
In order to avoid over 135 lines of code, we could seek another approach for the creation of the popup 🤓 Check out the following [link](https://wesbos.com/template-strings-html) so that you can find more concrete guidance. A quick example:

    const div = document.createElement('div');
    div.id = 'overlay';
    div.classList.add('overlay');
    div.innerHTML = '<img src="./images/buttons/Icon.png" class="cancelBttn"></img>'
    + '<ul class="mobile-menu-ul">'
    + '<li class="mobile-menu-li"><a href="#section-1">Porfolio</a></li>'
    + '<li class="mobile-menu-li"><a href="#section-2">About</a></li>'
    + '<li class="mobile-menu-li"><a href="#section-3">Contact</a></li>'
    + '</ul>';
    document.body.appendChild(div);

Please make the necessary changes so that the code follow JS best practices 😁

Reply:
Well done 🙌
- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.

Reply:
Kindly make the necessary changes so that your page doesn't enable scrolling when your popup is open. This will confuse and affect immensely the user experience of your app 📝

Reply:
resolved 👍